girder/utility/server.py

```diff
@@ -18,16 +18,34 @@
 ###############################################################################
 
 import cherrypy
+import mako
 import mimetypes
 import os
 import six
 
 import girder.events
 from girder import constants, logprint
+from girder.api.describe import API_VERSION
 from girder.utility import plugin_utilities, model_importer
 from girder.utility import config
 from . import webroot
 
+# TODO make a pretty error page someday.
+_errorTemplate = """
+<h1>${status}</h1>
+<p>${message}</p>
+<p><i>Girder server version ${version}</i></p>
```

Comment:
After some further thinking, I'd rather we not expose the server version in more places for now.  We may want to retreat from doing it more generally.

So I say let's remove it from here, I don't think it is urgent that we have it here regardless.